tests/console/commands/test_invalid_command.py

```diff
@@ -0,0 +1,14 @@
+import pytest
+
+from clikit.args import ArgvArgs
+
+from poetry.console.config import ApplicationConfig
+from poetry.exceptions import PoetryException
+
+
+def test_overwrite_exception_in_case_of_invalid_command(app):
```

Comment:
Any chance we can add a fil ltest to validate the `io` output does not contain the stack trace?

Reply:
I can add this check, but from my point of view it makes the test fragile.

I can check that the exception has `ex.__cause__ is None`. For us it means that the exception doesn't contain the original stacktrace from click